### DISCUSSION TITLE
Tab not working

### DISCUSSION THREAD

**Discussion by laique-lirctek** [community/community#11589] [Copilot Conversations]
i am getting the suggestion of github copilot, but i m not able to accept it. tab is not  working

**Comment by jansorg** [comment]
Thanks for reporting! Could you add which editor or IDE you're using?

**Comment by laique-lirctek** [comment]
I am using vs code

**Comment by matthewferry** [comment]
The same happens for me. I'm using vscode in Codespaces. Pretty sure it's because of my clj-kondo extension is using the tab keybinding aggressively. Is there any way to pick a different keybinding for Copilot autocomplete?

**Reply by matthewferry** [reply]
never mind, I see [this was already discussed](https://github.com/github/feedback/discussions/6919#discussioncomment-1553837)!

**Comment by achisholm** [comment]
I want to report the same issue on my first attempt to use Copilot. I am using VS Code on macOS.

I have disabled all extensions, and this issue persists for me.

**Comment by panckreous** [comment]
Same for me now. VS Code macos.
